src/api/index.ts

```diff
@@ -72,11 +72,10 @@ const fetchAnnouncements = async (andFilters: string[]): Promise<TrainAnnounceme
                 ${andFilters.join('')}
               </AND>
             </FILTER>
-            ${
-              Object.keys(trainAnnouncementModel)
-                .map(key=> `<INCLUDE>${key}</INCLUDE>`)
-                .join("\n")
-             }
+            ${Object.keys(trainAnnouncementModel)
+          .map(key => `<INCLUDE>${key}</INCLUDE>`)
+          .join("\n")
+        }
```

Comment:
NITPICK: was this change on purpose? I'm not going to have opinions on the code style in this repository, pointing out a formatting change that was maybe not intentional.

Reply:
good find, some kind of auto format did this